tests/drivers/build_all/led/app.overlay

```diff
@@ -31,5 +31,40 @@
 				label = "Test GPIO LED";
 			};
 		};
+
+
+		test_i2c: i2c@11112222 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			compatible = "vnd,i2c";
+			reg = <0x11112222 0x1000>;
+			label = "TEST_I2C_CTLR";
```

Comment:
Please remove these label properties when you get a chance. People are in the middle of a treewide cleanup to remove them.